### ISSUE TITLE
dead grafana link in project description

### ISSUE THREAD

**Issue by rapenne-s** [NixOS/ofborg#619]
the grafana link https://monitoring.nix.ci/dashboard/db/ofborg on the project description is not working

**Comment by cole-h** [comment]
Yeah, we used to have a real nice dashboard... But unfortunately, the machine hosting it died some time last year, we didn't have a backup of the dashboard, and I don't know enough about how to use Grafana to make another one. I just pay attention to Prometheus these days: https://nix.ci/prometheus

(Example query: https://nix.ci/prometheus/graph?g0.expr=ofborg_queue_evaluator_waiting&g0.tab=1&g0.stacked=0&g0.show_exemplars=0&g0.range_input=2h&g1.expr=ofborg_queue_evaluator_consumers&g1.tab=1&g1.stacked=0&g1.show_exemplars=0&g1.range_input=1h&g2.expr=ofborg_queue_builder_waiting%7Barch!~%22.*-lowprior%22%7D&g2.tab=1&g2.stacked=0&g2.show_exemplars=0&g2.range_input=6h&g3.expr=ofborg_queue_builder_in_progress%7Barch!~%22.*-lowprior%22%7D&g3.tab=1&g3.stacked=0&g3.show_exemplars=0&g3.range_input=1d&g4.expr=ofborg_queue_builder_consumers%7Barch!~%22.*-lowprior%22%7D&g4.tab=1&g4.stacked=0&g4.show_exemplars=0&g4.range_input=1d)

**Comment by rapenne-s** [comment]
If that can help, you could just use node-exporter on the server and use [the according grafana dashboard](https://grafana.com/grafana/dashboards/1860-node-exporter-full/) that display so many things. I [wrote about a setup with it on NixOS](https://dataswamp.org/~solene/2022-09-11-exploring-monitoring-stacks.html#_Setup_1:_Prometheus_server_scraping_remote_node_exporter), it's really easy to setup
